### ISSUE TITLE
Error installing Dtls 1.0.2 using PIP

### ISSUE THREAD

**Issue by veganjay** [rbit/pydtls#12]
I am receiving the following error message when trying to install using pip:
```
# pip install Dtls
Collecting Dtls
  Using cached Dtls-1.2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-XmO1Kb/Dtls/setup.py", line 53, in <module>
        long_description = open("README.md").read()
    IOError: [Errno 2] No such file or directory: 'README.md'
```
As far as I can tell, it looks like a packaging issue.

Also, this might be related, the install command encounters an error message:
```
# python setup.py install
running install
...
error: can't copy 'dtls_package_files': doesn't exist or not a regular file
```

**Comment by rbit** [comment]
Fixed in release 1.2.2. Thank you for reporting.

**Comment by veganjay** [comment]
Thanks for fixing this.  I successfully installed using pip.
